### ISSUE TITLE
Erreur 404 pour le pdf du module

### ISSUE THREAD

**Issue by deuzeffe** [MTES-MCT/parcours_r_socle_introduction#16]
Bonjour,

Ne pas hésiter à me ré-aiguiller si je ne suis pas au bon endroit.

J'aurais aimé récupérer le pdf du module 1 de façon à pouvoir lire hors-ligne mais https://mtes-mct.github.io/parcours_r_socle_introduction/D%C3%A9buter-avec-R-et-RStudio.pdf renvoie une erreur 404 :( 

Merci de ce que vous pourrez faire s'il y a quelque chose à faire.

**Comment by jdlom** [comment]
Oui, en effet. Les pdfs ne sont effectivement pas disponibles. Il faut qu'on prenne le temps de régler cela.

**Comment by jengelaere** [comment]
En attendant qu'on répare le bouton, une version à jour du support de cours est disponible ici : 
https://github.com/MTES-MCT/parcours_r_socle_introduction/blob/master/M1_Debuter_avec_R_et_RStudio.pdf

**Comment by deuzeffe** [comment]
@jengelaere merci !
